### PR TITLE
Fix #321 Specify the release name to sentry

### LINKED ISSUE #321: Specify the release name to sentry
We should include the current release w/ sentry's `ClientOptions`:

```
release: sentry::release_name!()
```

### PR DESCRIPTION
**Pull request by fzzzy**
Also update the Cargo.lock files so that building on macOS works again.

Please remember to consult our [contributing guidelines](https://github.com/mozilla-services/syncstorage-rs/blob/master/CONTRIBUTING.md#sending-pull-requests) before opening your PR.

- [x] **Title** begins with _type_ (fix, feature, doc, chore, etc) and a short description with no period
- [x] **Description**  outlines the change
- [ ] **Test cases** included in the change (if appropriate)
- [x] **Closes** or **Issue** link to associated issue(s)

### CODE REVIEW

**Comment by fzzzy** [comment]
Ok, reformatted the commit message.

File: src/main.rs [hunk 1]
```diff
@@ -39,6 +39,7 @@ fn main() -> Result<(), Box<dyn Error>> {
     };
     let sentry = sentry::init(sentry::ClientOptions {
         transport: Box::new(curl_transport_factory),
+        release: sentry::release_name!(),
```
Comment (tublitzed) [inline src/main.rs]: how does sentry get this information? Ie, is it the version number, or...?

Comment (pjenvey) [reply src/main.rs]: yup, it grabs them from the cargo compile time env vars, `CARGO_PKG_NAME`@`CARGO_PKG_VERSION`

Comment (tublitzed) [reply src/main.rs]: ah, perfect. thanks

**Pull request merged**